sdk/spring-sdk-testkit/src/test/java/spring/testmodels/actions/EchoAction.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package springsdk.testmodels.actions;
+package spring.testmodels.actions;
```

Comment:
I don't have a strong opinion about this but 'spring' here seems to me like it may not be precise. Meaning, it's not actually a `spring` component but a `kalix` one used in the 'springsdk` testkit. How about leaving them as they were or maybe `kalix.spring`?. or none of it and just merging it with Eduardo's blessings (and mine :)

Reply:
The notion of `springsdk` is being removed in favor of `javasdk` with a thin layer of Spring Boot integration. 

The choice to use `kalix.spring` and `kalix.spring.boot` is more related with a convention that we have seen in other projects. 

For that particular case, I think we can pick `kalix.spring.boot`. I will change it.

Reply:
@franciscolopezsancho, I see now that those tests are from the kalix-spring-testkit. The package should just be `kalix.javasdk.testmodels`.

Although the module still have `spring` in its name, the classes themselves are not Spring related.